01/Joshiraku 01 - Notes.ass

```diff
@@ -50,7 +50,7 @@ Comment: 0,0:00:00.00,0:00:10.03,Trivia,,0,0,0,kara,{\pos(131,623)}If you'd like
 Comment: 0,0:00:00.00,0:00:10.03,Trivia,,0,0,0,kara,{\pos(1029,762)}We hope you enjoy the show! - Kaleido Team
 Comment: 0,0:00:10.03,0:00:20.04,Trivia,,0,0,0,kara,{\pos(189,91)}Tetora is performing "rakugo", a classic \NJapanese comedic storytelling art, and \Nis also where the title comes from. {https://ja.wikipedia.org/wiki/%E8%90%BD%E8%AA%9E}
 Comment: 0,0:00:10.03,0:00:20.04,Trivia,,0,0,0,kara,{\pos(998,243)}These stories often contain a lot of \Nclever wordplay and exaggerated voices, \Nmaking them a pita to translate.
-Comment: 0,0:00:10.03,0:00:20.04,Trivia,,0,0,0,kara,{\pos(163,450)}The audience being so small likely indicates this \Nopening act was performed very early in the day.
+Comment: 0,0:00:10.03,0:00:20.04,Trivia,bkt,0,0,0,kara,{\pos(163,450)}The audience being so small likely indicates this \Nopening act was performed very early in the day.{bkt: Is this your own conjecture? It's probably not wrong considering they need time to have the rest of their shenanigans in the episode, but I don't know if I'd jump to this conclusion without more supporting info. There was a remark in one of the later episodes that rakugo wasn't as popular as it used to be.}
```

Comment:
This was conjecture from the JP wiki. While rakugo isn't as popular anymore, usually it wouldn't be *this* empty, I think? I don't mind dropping it because it is pretty conjecture-y